src/subscription_manager/managercli.py

```diff
@@ -2262,7 +2270,9 @@ def _do_command(self):
                         else:
                             data['management_enabled'] = _("No")
 
-                        print columnize(AVAILABLE_SUBS_LIST, _none_wrap,
+                        kwargs = {"filter_string": self.options.filter_string,
+                                  "match_columns": AVAILABLE_SUBS_MATCH_COLUMNS}
+                        print columnize(AVAILABLE_SUBS_LIST, _highlight_by_filter_string,
```

Comment:
(Not related to this pr).

At some point, I'd like to ditch columnize's odd usage of args/kwargs (and it's callbacks that also have to do odd things). I don't see any particular reason we could just pass a 'columns' list in instead of *args.